test/pipelines-it-installation.yml

```diff
@@ -0,0 +1,49 @@
+jobs:
+- job: 'pip_install_ubuntu_python36'
+  pool:
+    vmImage: 'ubuntu-18.04'
+  strategy:
+    matrix:
+      Python36:
+        PYTHON_VERSION: '3.6'
+
+  steps:
+  - script: |
+      python3 -V
+      python3 -m pip install --upgrade pip setuptools --user
+      python3 -m pip install --upgrade nni --user
```

Comment:
This only tests released version right?

Reply:
yes.

Reply:
So when this test case fails, we have already published a broken release...

Reply:
This is just a start point, we can add more later, for example:
1. pre-release testpypi test
2. make dev install
3. install for different platforms (python version, virtual envs)
4. anything related installation not covered by IT.